docs/yaml_output.md

```diff
@@ -0,0 +1,45 @@
+# YAML Format Output
+
+You can get the results in a YAML file to be consumed by a downstream tool or script.
+
+`$ tern -l report -f yaml -i golang:1.12-alpine -o output.yaml`
+
+A yaml foramt output file starts with line comments indicating the version of Tern. It has a `image` key(image.py) corresponding to the analyzed image. Here are the subkeys and their description:
+
+- `config`: the image config metadata
```

Comment:
@nishakm do we want to list the subkeys of the `config` key similar to how we do with `packages` below? There are a few interesting ones like `architecture`, `created` and `os`.

Reply:
On second thought, this might be a lot of work as there are lots of other subkeys with metadata as well :)